wgpu-native/src/instance.rs

```diff
@@ -67,6 +63,74 @@ pub extern "C" fn wgpu_instance_create_surface_from_winit(
         .register(surface)
 }
 
+#[allow(unused)]
+#[no_mangle]
+pub extern "C" fn wgpu_instance_create_surface_from_macos_layer(
```

Comment:
any reason this entry point isn't hidden for non-macos targets?

Reply:
To unconditionally include in wgpu.h for now, there are other ways we could work around it though